### PR TITLE
Kerberos: support credential request and check

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: src/main/perl/Kerberos.pm [hunk 1]
```diff
@@ -314,6 +319,18 @@ sub get_context
...
 
+    my $cred;
+
+    if ($opts{usecred}) {
+        $self->verbose("trying to get credential");
+        $cred = $self->get_cred(name => $name);
+        # Already reports error
+        return if ! $cred;
+    } else {
+        $self->verbose("No checking for credential");
```
Comment (ned21) [inline src/main/perl/Kerberos.pm]: This adds a message to every invocation if not using the new option. Should this be debug or is it important enough to be a warning? Also, "No" -> "Not" ?

Comment (stdweird) [inline src/main/perl/Kerberos.pm]: i'm not sure. the context mechanism is used to get the initial tokens, but without passing a credential instance, the principal passed via the name instance is used in a sort of best-effort way.

Comment (ned21) [inline src/main/perl/Kerberos.pm]: OK, let's leave it as verbose then. Can the messages be made consistent?
"trying to get credential" -> "Trying to get credential".
"No checking for credential" -> "Not checking for credential"

**Comment by stdweird** [comment]
@ned21 addressed remark

**Pull request merged**